src/lib/tabs/tab-body.ts

```diff
@@ -113,7 +114,17 @@ export class MatTabBodyPortal extends CdkPortalOutlet implements OnInit, OnDestr
     'class': 'mat-tab-body',
   },
 })
-export class MatTabBody implements OnInit {
+export class MatTabBody implements OnInit, OnDestroy {
+
+  /** Current position of the tab-body in the tab-group. Zero means that the tab is visible. */
+  private _positionIndex: number;
+
+  /** Subscription to the directionality change observable. */
+  private _dirChangeSubscription: Subscription = Subscription.EMPTY;
```

Comment:
The `Subscription` type here is inferred already.

Reply:
Makes sense.